README.md

```diff
@@ -35,10 +35,14 @@ docker run -d \
 This makes a set of self signed certificates and minimal configuration to
 run server.
 
-Please note that the generated certificated will have their `CN` set to `localhost`. In order to modify the parameters used for the certificate generation:
```

Comment:
Please change this section to this:

```
## Environment variables

Certificate attributes can be customized using environment variables.

| Variable | Default value |
| --- | --- |
| `CERT_BITS` | 4096 |
| `CERT_EXPIRATION_DAYS` | 365 |
| `CERT_ORGANIZATION` | "Göteborg Bit Factory" |
| `CERT_CN` | localhost |
| `CERT_COUNTRY` | SE |
| `CERT_STATE` | "Västra Götaland" |
| `CERT_LOCALITY` | "Göteborg" |

Note that, by default, the generated certificates will have their `CN` set
to `localhost`.

To regenerate certificates or modify their parameters:
- Delete everything in `/pki/` except the generate scripts (`generate*`) and the `vars` file.
- Run (modify the variables to what you need)
  ```sh
  docker exec -t -i \
    -e CERT_BITS=4096 \
    -e CERT_EXPIRATION_DAYS=365 \
    -e CERT_ORGANIZATION="Göteborg Bit Factory" \
    -e CERT_CN=localhost \
    -e CERT_COUNTRY=SE \
    -e CERT_STATE="Västra Götaland" \
    -e CERT_LOCALITY="Göteborg" \
    <container-id> /var/taskd/pki/generate
  ``` <remove this, it's here because otherwise the markdown will be broken>
```

Reply:
Remember to remove the text at the end and leave only the three inverted quotation marks ```.